test/e2e/main_test.go

```diff
@@ -92,7 +92,7 @@ func testMain(m *testing.M) int {
 	flag.BoolVar(&testOptions.flowVisibility, "flow-visibility", false, "Run flow visibility tests")
 	flag.BoolVar(&testOptions.deployAntrea, "deploy-antrea", true, "Deploy Antrea before running tests")
 	flag.StringVar(&testOptions.coverageDir, "coverage-dir", "", "Directory for coverage data files")
-	flag.StringVar(&testOptions.skipCases, "skip", "", "Key words to skip cases")
+	flag.StringVar(&testOptions.skipCases, "skipCases", "", "Key words to skip cases")
```

Comment:
should probably be `skip-cases` for consistency with the other flags (`linuxVMs` and `windowsVMs` should probably also have been `linux-vms` and `windows-vms`, but it's too late now...).

Reply:
sure